src/odemis/driver/technolution.py

```diff
@@ -443,18 +533,221 @@ def _mergeMetadata(self):
         return md
 
     def _setAcqDelay(self, delay):
+        """
+        Setter which checks if detector can record images before ebeam scanner has started to scan.
```

Comment:
setter of what?
Add that this is the delay before starting the recording with the detector. And then add that it checks....